gulpfile.cjs

```diff
@@ -921,6 +921,7 @@ gulp.task("test", function (done) {
   var release = argv.release ? argv.release : false;
   var failTaskOnError = argv.failTaskOnError ? argv.failTaskOnError : false;
   var suppressPassed = argv.suppressPassed ? argv.suppressPassed : false;
+  global.tdd = argv.tdd ? argv.tdd : false;
```

Comment:
I don't think setting this global is necessary, you should be able to just pass the options to `new Karma.Server(` and that overrides the defaults in karma.conf

Reply:
@mramato I changed both the coverage and the tests definitions.